engines/router/missionctl/config/config.go

```diff
@@ -36,6 +36,8 @@ const (
 	ConsoleLogger ResultLogger = "CONSOLE"
```

Comment:
note: console logger is only removed from API, the implementation isn't remove as it can be potentially useful for debugging